crates/holochain_types/src/app.rs

```diff
@@ -406,7 +406,7 @@ impl From<RunningApp> for InstalledApp {
     }
 }
 
-/// An app which is either Paused or Disabled, i.e. not Running
+/// An app which is Disabled, i.e. not Running
```

Comment:
If this is related to `AppStatus::Disabled`, to help users navigate the documentation, you could change to "[`AppStatus::Disabled`]".